src/main.ts

```diff
@@ -182,7 +184,12 @@ export async function run(): Promise<void> {
   } else {
     const allPatterns = filePatterns
       .split('\n')
-      .filter(p => !DEFAULT_EXCLUDED_FILES.includes(p) && p !== '')
+      .filter(
+        p =>
+          !DEFAULT_EXCLUDED_FILES.map(
+            p => `!${path.join(workingDirectory, p.replace(/^!/, ''))}`
```

Comment:
🚫 **[eslint]** <[no-shadow](https://eslint.org/docs/rules/no-shadow)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'p' is already declared in the upper scope on line 188 column 9.